docs/ref/contrib/gis/gdal.txt

```diff
@@ -1832,3 +1832,25 @@ Settings
 A string specifying the location of the GDAL library.  Typically,
 this setting is only used if the GDAL library is in a non-standard
 location (e.g., ``/home/john/lib/libgdal.so``).
+
+Exceptions
+==========
+
+.. exception:: GDALException
+
+    The base GDAL exception, indicates a GDAL-related error.
+
+    .. note::
+
+        ``OGRException`` is a legacy name for this exception only kept for
+        backwards compatibility.
+
+.. exception:: OGRIndexError
+
+    An exception raised when an invalid result is returned from the underlying
+    library.
+
+.. exception:: SRSException
+
+    An exception raised when an error occurs when constructing or using a
```

Comment:
Please wrap at 79 characters.

Reply:
This line is wrapped round at 73 with the next word being "spatial".